mllib/src/main/scala/org/apache/spark/ml/param/shared/sharedParams.scala

```diff
@@ -154,7 +154,7 @@ private[ml] trait HasVarianceCol extends Params {
 }
 
 /**
- * Trait for shared param threshold (default: 0.5).
```

Comment:
The definition in ```SharedParamsCodeGen``` has no default value yet, since both ```LogisticRegression``` and ```LinearSVC``` extends from ```HasThreshold```, but they have different default value.
```
ParamDesc[Double]("threshold",
        "threshold in binary classification prediction, in range [0, 1]",
        isValid = "ParamValidators.inRange(0, 1)", finalMethods = false, finalFields = false),
```